src/main.js

```diff
@@ -141,9 +145,15 @@ bot.on("messageCreate", async msg => {
   if (msg.guildID) return; // Ignore messages sent to a guild
 
   const isBlocked = await blocked.isBlocked(msg.author.id);
+  const isDisabled = await snippet.get("isDisabled");
 
   if (isBlocked) return;
-  if (msg.content.length > 1900) return bot.createMessage(msg.channel.id, `Your message is too long to be recieved by Dave. Please shorten it! (${msg.content.length}/1900)`);
+  if (isDisabled && isDisabled.body.toLowerCase().includes("disabled")) {
+    let disabledSnippet = await snippet.get("disabled");
+    let disabledMessage = disabledSnippet ? disabledSnippet.body : "ModMail is currently unavailable while the server is closed.\n If there is an outage please check <#535189342898094081> for updates.";
+    return utils.sendInfo(msg, disabledMessage);
+  }
```

Comment:
Is it a good idea to prevent people with open threads already from also messaging in their respective threads? If this was decided upon then fair enough

Reply:
Discussed